scripts/start-deployMagma

```diff
@@ -8,18 +8,14 @@ isDebugging && set -x
 
 resolveVersion
 
-if ! downloadUrl=$(get --json-path '$.link' "https://api.magmafoundation.org/api/v2/${VERSION}/latest/${MAGMA_VERSION}"); then
-  logError "Failed to locate latest Magma download for ${VERSION}. Is that version supported?"
-  exit 1
-fi
-
-if [[ $downloadUrl == null ]]; then
-  logError "Magma does not seem to be available for $VERSION"
-  exit 1
+# Neo Magma currently supports just 1.21.x
+if [[ ! $VERSION =~ ^1\.21(\.[0-9]+)?$ ]]; then
```

Comment:
Since bash has prefix matching, use this since regex is fun, but aways hard to maintain:


```suggestion
if [[ ! $VERSION = 1.21.* ]]; then
```